app/views/users/_search.html.slim

```diff
@@ -2,7 +2,7 @@
   .form-group
     = text_field_tag :search, params[:search], placeholder: "Search user or team", class: "form-control"
   .form-group
-    = submit_tag "Go", class: "form-control btn btn-success"
+    = submit_tag "Search", class: "form-control btn btn-default"
```

Comment:
UX nitpicky, but: I called the button 'Go' because the directive "Search" is already in the placeholder. If you change the button text to 'Search', I would change the placeholder as well.